src/console/controllers/ProjectConfigController.php

```diff
@@ -64,11 +83,82 @@ public function options($actionID): array
                 break;
             case 'diff':
                 $options[] = 'invert';
+                // no break
```

Comment:
Why would `diff` need the `fromFile`, `message`, `updateTimestamp`, and `force` options?

Reply:
I just mistakenly left out the break, but ecs added the `no break` comment, making it seem intentional 🤦